openapi/components/schemas/AML.yaml

```
@@ -0,0 +1,150 @@
+type: object
+properties:
+  firstName:
+    type: string
+    description: First Name
+  lastName:
+    type: string
+    description: Last Name
+  source:
+    type: string
+    readOnly: true
+    description: Which list this came from.
+  sourceType:
+    readOnly: true
+    description: The list type
+    type: string
+    enum:
+      - pep
+      - sanctions
+      - adverse-media
+  gender:
+    type: string
+    readOnly: true
+    nullable: true
+    description: Gender of returned identity (if `type` is `individual`)
+  title:
+    type: array
+    readOnly: true
+    nullable: true
+    description: The identity's title of their position, i.e. `President` or `General`
```

```suggestion
    description: The identity's title of their position.
    example: President
```
I would make an example.  This is another line like so (if GitHub let's me add a line).